packages/@divvi/mobile/src/qrcode/QRScanner.tsx

```diff
@@ -94,20 +142,16 @@ export default function QRScanner({ onQRCodeDetected }: QRScannerProps) {
   )
 
   return (
-    <RNCamera
-      style={styles.camera}
-      type={RNCamera.Constants.Type.back}
-      onBarCodeRead={onQRCodeDetected}
-      barCodeTypes={[RNCamera.Constants.BarCodeType.qr]}
-      flashMode={RNCamera.Constants.FlashMode.auto}
-      captureAudio={false}
-      autoFocus={RNCamera.Constants.AutoFocus.on}
-      // Passing null here since we want the default system message
-      // @ts-ignore
-      androidCameraPermissionOptions={null}
-      notAuthorizedView={<NotAuthorizedView />}
-      testID={'Camera'}
-    >
+    <View>
```

Comment:
Avoid using CameraView as a parent, which is generally discouraged: https://github.com/expo/expo/issues/31132#issuecomment-2307177725.